New Features for 2025-06-01-preview API version:

melissamserv opened the pull request:
New Features for **2025-06-01-preview** API version:

-  Support for URIs and IP addresses in SAN properties: 
   Add the **uris** and **ipAddresses** attributes to **Subject Alternative Names** for certificate requests.

-  PFX to PEM conversion in Get Certificate:
   Add the **outContentType** query parameter to **Get Certificate** operation to specify the format of the certificate in the response.

- Include the version of the previous certificate in secret responses:
  Add the **previousVersion** attribute to the **SecretBundle** in the response.

melissamserv committed:
Copy files from stable/7.6

Copied the files in a separate commit.
This allows reviewers to easily diff subsequent changes against the previous spec.

melissamserv committed:
Update version to 2025-06-01-preview

Updated the API version from 7.6 to 2025-06-01-preview.

melissamserv committed:
Add tag for 2025-06-01-preview in readme file

melissamserv committed:
Copy TSP examples from 7.6 to 2025-06-01-preview

Copied the files in a separate commit.
This allows reviewers to easily diff subsequent changes against the previous version.

melissamserv committed:
Update TSP examples version to 2025-06-01-preview

Updated the API version from 7.6 to 2025-06-01-preview.

melissamserv committed:
Add 2025-06-01-preview API version enum value to TSP

melissamserv committed:
Remove unnecessary backticks in v2025_06_01_preview enum

melissamserv committed:
Add uris and ipAddresses attributes to SubjectAlternativeNames

melissamserv committed:
Add previousVersion attribute

melissamserv committed:
Add outContentType parameter

melissamserv committed:
Improve descriptions of the previousVersion attribute and outContentType parameter

heaths commented:
Isn't this a dup of #35534? If so, please close dups so we're not looking at both.

melissamserv committed:
Fix TSP validation for SecurityDomain

Add new 2025-06-01-preview API version to SecurityDomain spec
Copy TSP SecurityDomain examples from 7.6 to 2025-06-01-preview and update the API version

melissamserv committed:
Improve description of outContentType parameter
